libs/cloud/odc/aws/queue.py

```diff
@@ -180,3 +181,30 @@ def get_messages(
         raise Exception(f"Limit {limit} is not valid.")
 
     return itertools.islice(messages, limit)
+
+
+def publish_to_topic(topic_name: str, action: str, stac: dict):
+    """
+    Publish 'added' or 'archived' action to the provided sns topic
+    """
+    sns = boto3.client("sns")
+    sns_topic = sns.create_topic(Name=topic_name)
+
+    stac = json.dumps(stac)
+    message = {
+        "action": action,
+        "stac": stac,
+    }
+    sns.publish(
+        TopicArn=sns_topic.get("TopicArn"),
+        Message=json.dumps(message),
+        MessageAttributes={
+            "action": {
+                "DataType": "String",
+                "StringValue": "ARCHIVED {or} ADDED",
+            },
+            "stac": {
```

Comment:
We don't want to include the STAC doc inside the message attributes - that's the message payload

Reply:
Fixed